### ISSUE TITLE
Matcher for verifying `creates_unique` on relationships

### ISSUE THREAD

**Issue by sean-wedig** [sineed/neo4j-rspec#9]
`ActiveRel.creates_unique` can be used to ensure that a relationship is only created once per set of properties.  In our domain, this is fairly common to apply to `ActiveRel` models, and it would be convenient to have a matcher that verified this succinctly.  It would need to support verifying the argument passed to `create_unique` (i.e, `:none`, `:all`, `{on: [keys]}`).

**Comment by sineed** [comment]
For now you can use `rspec` syntax for it:

``` ruby
expect(ActiveRel).not_to be_unique
ActiveRel.creates_unique
expect(ActiveRel).to be_unique
```

I will think about checking values passed throught `creates_unique` method.

**Comment by sineed** [comment]
I released 0.2.2 version with `use_create_unique` matcher
